entities/communication/voiceCallSetting.json

```diff
@@ -0,0 +1,99 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "The voice call settings unique identifier."
+        },
+        "staff_uid": {
+            "type": "string",
+            "description": "Receiving forwarded call staff uid."
+        },
+        "forward_number": {
+            "type": "string",
+            "description": "The phone number of the staff receiving the forwarded call."
+        },
+        "dedicated_number": {
+            "type": "string",
+            "description": "The dedicated number of the business receiving calls from clients."
+        },
+        "forwarding_enabled": {
+            "type": "boolean",
+            "description": "Is forwarding the voice call enabled or not."
+        },
+        "call_forwarding_policy": {
+            "type": "enum",
+            "values": [
+                "ALWAYS",
+                "WORK_HOURS",
+                "NEVER"],
+            "description": "When to forward the call based on staff availability."
+        },
+        "staff_weekly_availability_uid": {
+            "type": "string",
+            "description": "Staff weekly availability details uid."
```

Comment:
I don't understand what this is doing? is this a reference to a value object. What do we have in that value object. How does it impact functionality?